src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -1320,6 +1327,7 @@
     <Resource Include="UI\Images\icon-whats-new-small.png" />
     <Resource Include="UI\Images\icon-dictionary-small.png" />
   </ItemGroup>
+  <ItemGroup />
```

Comment:
This line seems not needed?

Reply:
@QilongTang This line was removed in the next commit: https://github.com/DynamoDS/Dynamo/pull/11534/commits/eb0d517fb2dd151c02a9c68afc082913668a6371